content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+
+The Istio network API defines four configuration objects: gateways, virtual
+services, destination rules, and service entries.
+The diagrams on this page don't show data plane traffic. Unless otherwise
+indicated, the arrows in the diagrams portray the traffic routes that the
+network configuration objects configured using destination rules. The diagrams
+show the possible control plane connections between the Envoy proxies of the
+network configuration objects.
+
+The following diagram shows the **configurable traffic routes** that the
+network configuration objects allow for a basic mesh with two different
+versions of a service:
+
+{{< image width="80%"
+    link="./net-config-1.svg"
+    caption="Istio network configuration objects overview"
+    >}}
+
+This document describes the key configurations each configuration object
+enables. Additionally, this document provides high-level examples of how the
+Istio network API can help you handle internal, incoming, and outgoing traffic.
+Lastly, the document discusses the benefits of using the Istio network API for
+your cloud applications.
+
+# The Istio network configuration objects
+
+The Istio network configuration objects provide you with four objects to
+configure the traffic routes between your services and your external resources:
+virtual services, destination rules, service entries, and gateways. You can add
+and configure these objects in a variety of ways to suit your needs. Each
+configuration object serves a specific purpose and provides specific
+configurations for your Istio service mesh. To create a solution for your
+specific mesh, you can combine the configuration objects in a way that best
+fulfills your needs.
+
+You can define the Istio network configuration objects as Kubernetes Custom
+Resource Definition [(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+objects and store their configuration in YAML files. The following sections
+describe the purposes and specific configurations of each configuration object
+and provide examples.
+
+## Virtual Services
+
+You can use virtual services for fine-grained traffic control. Istio can use
+virtual services to configure  ingress, canary, traffic rewrite, or traffic
+policing. A virtual service is not required for simple connectivity. A service
+can map one or more user-addressable destinations to the actual destination
+workloads inside the mesh without a virtual service. The Istio virtual services
+can configure traffic routes to:
+
+-  Other virtual services in the mesh.
+-  Specific services or services' subsets in the mesh.
+-  Other network configuration objects in the mesh.
+
+{{< image width="80%"
+    link="./net-config-2.svg"
+    caption="Possible configurations between virtual services and workloads"
+    >}}
+
+As you can see, virtual services are very flexible and can accommodate a wide
+variety of topologies. With destination rules, virtual services configure the
```

Comment:
Destination rules are not required

Reply:
Understood. What functionality would you say a destination rule can add to a virtual service? What relationships are possible between the two?